MAUI/Popup/popup-positioning.md

```diff
@@ -264,4 +264,300 @@ public class ViewModel : INotifyPropertyChanged
 {% endhighlight %}
 {% endtabs %}
 
-![Displaying a .NET MAUI Popup at Relative position using MVVM.](Images/popup-positioning/maui-popup-relative-positioning-in-mvvm.png)
\ No newline at end of file
+![Displaying a .NET MAUI Popup at Relative position using MVVM.](Images/popup-positioning/maui-popup-relative-positioning-in-mvvm.png)
+
+## How to
+
+### Load the SfPopup in CellTappedEvent of the SfDataGrid
+
+The `SfPopup` allows opening it in the [CellTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellTapped) event of the `SfDataGrid`.
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="14" %}
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:sfDatagrid="clr-namespace:Syncfusion.Maui.DataGrid;assembly=Syncfusion.Maui.DataGrid"
+             xmlns:sfPopup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+             xmlns:local="clr-namespace:PopupMauiPositioning"
+             x:Class="PopupMauiPositioning.MainPage">
+  <ContentPage.BindingContext>
+    <local:ViewModel x:Name="viewModel" />
+  </ContentPage.BindingContext>  
+  <ContentPage.Content>
+    <Grid>
+      <sfDatagrid:SfDataGrid x:Name="dataGrid"
+                             ItemsSource="{Binding OrdersInfo}"
+                             CellTapped="OnDataGridCellTapped"    
+                             ColumnWidthMode="Fill">
+      </sfDatagrid:SfDataGrid>
+      <sfPopup:SfPopup x:Name="sfPopup" 
+                       HeaderTitle="Popup" 
+                       AutoSizeMode="Height"
+                       ShowCloseButton="True">
+        <sfPopup:SfPopup.ContentTemplate>
+          <DataTemplate>
+            <Label Text="A pop-up is a graphical user interface display area that suddenly appears in the foreground of the visual interface. Pop-up can be initiated by single or double tap or can simply be timed to occur. A pop-up window should be smaller than the background window or interface; otherwise, its a replacement interface."                               

```

Comment:
<Label Text= "A pop-up is a graphical user interface display area that suddenly appears in the foreground of the visual interface. Pop-up can be initiated by single or double tap or can be timed to occur. A pop-up window should be smaller than the background window or interface; otherwise, it's a replacement interface."